libs/model/src/contest/Contests.projection.ts

```diff
@@ -49,35 +60,77 @@ async function updateOrCreateWithAlert(
     );
 
   const { ticker, decimals } =
-    await protocol.contractHelpers.getTokenAttributes(url, contest_address);
+    await protocol.contractHelpers.getTokenAttributes(contest_address, url);
 
-  const [updated] = await models.ContestManager.update(
-    {
-      interval,
-      ticker,
-      decimals,
-    },
-    { where: { contest_address }, returning: true },
-  );
-  if (!updated) {
-    // when contest manager metadata is not found, it means it failed creation or was deleted
-    // here we are alerting admins and creating a default entry
-    const msg = `Missing contest manager [${contest_address}] on namespace [${namespace}]`;
-    log.error(msg, new MissingContestManager(msg, namespace, contest_address));
-
-    if (mustExist(`Community with namespace: ${namespace}`, community))
-      await models.ContestManager.create({
-        contest_address,
-        community_id: community.id!,
+  await models.sequelize.transaction(async (transaction) => {
```

Comment:
@rbennettcw let's talk about this transaction... just need to understand who's the aggregate here and the consistency boundaries we are protecting

Reply:
Two things were added to the XXXDeployed handlers:
- The creation of the `Contest` (because the `ContestStarted` event doesn't actually happen when contest initially starts– Ian verified this)
- The creation of the EVM event sources to start listening to events from the new contest contract

All of those things must happen for subsequent events to process correctly– and therefore should happen atomically.